### ISSUE TITLE
prior_predict() function?

### ISSUE THREAD

**Issue by bnicenboim** [paul-buerkner/brms#995]
Hi Paul,
Maybe it would be nice to add a `prior_predict()` function that refits the model with "sample_prior = "only" and then run `posterior_predict()`. I could add it, if you want.

**Comment by paul-buerkner** [comment]
Hmm, I have to think of that more. The problem is that this will cause a model refit, every time `prior_predict` is called which seems wasteful especially since users may also want to use other post-processing methods, such as, for example, `posterior_epred`, and we probably do not want to make `prior_` versions of all these methods where each time we trigger a model refit, even if it was with only the priors. What problem due you see in updating the model manually with `sample_prior = "only"` and then running `posterior_predict`?

**Comment by bnicenboim** [comment]
yeah, that's what I'm doing. I'm about to teach a course now, and it just seems weird to use "posterior" predict.

**Comment by paul-buerkner** [comment]
I see. I am however not sure that this is a sufficient reason for adding this extra functionality given my current concerns above.

**Comment by bnicenboim** [comment]
yeah, ok, I though just about having sample_prior = "only" and posterior_predict together in a more transparent function. I didn't think about the model refit... Feel free to close then.
